modules/openapi-generator/src/main/resources/php-nextgen/ObjectSerializer.mustache

```diff
@@ -316,20 +317,33 @@ class ObjectSerializer
     }
 
     /**
-     * Take value and turn it into a string suitable for inclusion in
+     * Take value and turn it into an array suitable for inclusion in
      * the http body (form parameter). If it's a string, pass through unchanged
      * If it's a datetime object, format it in ISO8601
      *
-     * @param string|\SplFileObject $value the value of the form parameter
+     * @param string|bool|array|DateTime|ArrayAccess|\SplFileObject $value the value of the form parameter
      *
-     * @return string the form string
+     * @return array [key => value] of formdata
      */
-    public static function toFormValue(string|\SplFileObject $value): string
-    {
+    public static function toFormValue(
```

Comment:
what about adding a test or 2 for this function (toFormValue) in https://github.com/OpenAPITools/openapi-generator/blob/master/samples/client/petstore/php/OpenAPIClient-php/tests/ObjectSerializerTest.php ?